include/fx/Echo.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+/**
+ * @file Echo.h
+ * @ingroup fx
+ * The Echo class.
+ */
+#include "fx/Effect.h"
+#include <vector>
+AUD_NAMESPACE_BEGIN
+class AUD_API Echo : public Effect
+{
+private:
+    double m_delaySec;      /* Delay time in seconds */
+    double m_feedback;      /* Feedback amount */
+    double m_mix;           /* Wet/dry mix */
```

Comment:
Is there a reason you chose `double` as a datatype for these variables? I think single precision `float` should be more than enough here and again it would be more consistent with the rest of the engine and like more performant!